resume-buddy/src/main/java/com/google/sps/api/Email.java

```diff
@@ -0,0 +1,73 @@
+package com.google.sps.api;
+
+import com.google.api.client.util.Base64;
+import com.google.api.services.gmail.Gmail;
+import com.google.api.services.gmail.model.Message;
+import com.google.sps.Consts;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+/** Class responsible to create an email and send it to recipient */
+public class Email {
+
+  private Email() {};
+
+  public static void sendEmail(String to, String subjectLine, String text) {
+    try {
+      Gmail service = GmailService.getGmailService();
+      MimeMessage mimeMessage = createEmail(to, Consts.FROM, subjectLine, text);
+      Message message = createMessageWithEmail(mimeMessage);
+      message = service.users().messages().send(Consts.FROM, message).execute();
+
+    } catch (Exception e) {
+      System.err.println("Could not send the email");
+    }
+  }
+
+  /**
+   * Create a MimeMessage using the parameters provided.
```

Comment:
Nit: here and on other functions. This should read "Creates..."

See https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment